database/migrations/2023_06_01_193901_create_table1.php

```diff
@@ -11,7 +11,8 @@
      */
     public function up(): void
     {
-        Schema::create('table', function (Blueprint $table) {
+        Schema::create('table1', function (Blueprint $table) {
```

Comment:
what is the table with 'table1' naming? You must already have the default users table, use it.
Otherwise, rename this table to 'users' and run:
`php artisan migrate:fresh`